src/content/chapter1_functions/lesson10_deprecations/en.html

```diff
@@ -4,11 +4,11 @@
   attribute.
 </p>
 <p>
-  If a deprecated function is reference the compiler will emit a warning,
+  If a deprecated function is used the compiler will emit a warning,
```

Comment:
```suggestion
  If a deprecated function is referenced the compiler will emit a warning,
```
It's when the value is referenced rather than used. You'll get the warning even if you never use it